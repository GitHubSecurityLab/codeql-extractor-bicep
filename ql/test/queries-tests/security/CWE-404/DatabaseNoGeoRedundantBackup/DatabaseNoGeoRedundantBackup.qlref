security/CWE-404/DatabaseNoGeoRedundantBackup.ql
