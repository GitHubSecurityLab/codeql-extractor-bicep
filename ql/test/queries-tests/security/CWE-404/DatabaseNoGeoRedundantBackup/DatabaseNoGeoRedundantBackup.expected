| app.bicep:10:13:13:5 | Backup | Database 'vulnerable-postgresql-no-geo' does not have geo-redundant backup enabled, which may result in data loss during regional disasters. |
| app.bicep:24:13:27:5 | Backup | Database 'vulnerable-mysql-disabled-geo' does not have geo-redundant backup enabled, which may result in data loss during regional disasters. |
| app.bicep:38:13:41:5 | Backup | Database 'vulnerable-mariadb-no-geo' does not have geo-redundant backup enabled, which may result in data loss during regional disasters. |
